packages/salesforcedx-vscode-lwc/README.md

```diff
@@ -1,5 +1,56 @@
-# LWC for Visual Studio Code
+# Lightning Web Components for Visual Studio Code
 
-salesforce-vscode-lwc is part of a private pilot for testing new features. We are not accepting pilot enrollments at this time. This extension will not work unless you are part of the pilot.
+This extension provides code-editing features for the Lightning Web Components programming model, which is part of the Lightning Component framework. It uses the default HTML language server from VS Code to provide syntax highlighting, code completion, an outline view of your files, and a Salesforce Lightning Design System (SLDS) linter.
 
-We provide salesforce-vscode-lwc to selected customers through a pilot program that requires agreement to specific terms and conditions. salesforce-vscode-lwc is subject to change and isn’t generally available unless or until Salesforce announces its general availability in documentation or in press releases or public statements. We can’t guarantee general availability within any particular time frame or at all. Make your purchase decisions only on the basis of generally available products and features.
+**Currently, this extension is not included in the [Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode).**
+
+## Prerequisites
+
+Before you set up this extension
+
+* Make sure that you have [Visual Studio Code](https://code.visualstudio.com/download) v1.26 or later.
+
+* Install the [Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) to get all the required dependencies.
+
+## Features Provided by This Extension
+
+* Notification of HTML and JavaScript file errors or compiler warnings
+
+* ESLint configuration for Lightning web components
+    * Errors and warnings appear in JavaScript files
+    * Hover over code to display warning messages
+    * Click the displayed message for available code actions
+  
+* Auto-completion for resources in JavaScript files
+    * Static resources
+    * Custom label imports
+    * Lightning web components imports from `engine`
```

Comment:
@dj-sf-git-it - This is not right. It should be `lwc` not `engine`. I believe that `engine` is now deprecated from reading the sample code.

Reply:
Correct! Thanks for catching that. I simply reused the feature list from the older version. I've updated and committed the change.